### DISCUSSION TITLE
How can I convert `great_tables` output in a Jupyter notebook into something that Quarto will render correctly in PDF?

### DISCUSSION THREAD

**Discussion by bzm3r** [quarto-dev/quarto-cli#10703] [Q&A]
### Description

Here is the raw notebook I am working with (you can paste it in a file called `example.ipynb`: 

It uses `polars`  to create a dataframe, and `great_tables` to output that dataframe with some highlighting. I render this notebook using Quarto, with the built-in PDF engine.

````json
{
 "cells": [
  {
   "cell_type": "raw",
   "metadata": {
    "vscode": {
     "languageId": "raw"
    }
   },
   "source": [
    "---\n",
    "title: \"Tables\"\n",
    "format:\n",
    "  pdf:\n",
    "    toc: true\n",
    "jupyter: python3\n",
    "---\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import polars as pl\n",
    "from great_tables import style, loc\n",
    "\n",
    "df = pl.DataFrame(pl.Series(\"x\", [0, 1, 2, 3]))\n",
    "\n",
    "df.style.tab_style(\n",
    "    style.fill(\"orange\"),\n",
    "    loc.body(\n",
    "        rows=pl.col(\"x\") == pl.col(\"x\").max(),\n",
    "    ),\n",
    ")\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
````

How can I show the formatted table properly in the output Quarto PDF?

**Comment by mcanouil** [comment]
You can't as HTML is not LaTeX.
Note that this is not a Quarto question.
You need to look for `great_tables` documentation.
If you emit HTML table, it will be read in the abstract format then emitted as LaTeX, but LaTeX is very different from HTML, so you will almost never get the same thing and not tools can change or will change that.

**Comment by gordonwoodhull** [comment]
Quarto does transfer some table styles from HTML/CSS to Typst, see

https://quarto.org/docs/blog/posts/2024-07-02-beautiful-tables-in-typst/index.html

So that is one route to showing a formatted table in a PDF.

However, the same translation does not currently exist from HTML/CSS to Latex. So you will lose the formatting when outputting to PDF via Latex.
